scripts/release/rpm/ubi9.dockerfile

```diff
@@ -0,0 +1,26 @@
+# Red Hat Universal Base Image 9: https://catalog.redhat.com/software/containers/ubi9/ubi/615bcf606feffc5384e8452e
```

Comment:
Can we reuse `ubi8.dockerfile`?

Reply:
Thanks, I'll make devel package be an arg.

In RHEL9, the valid names are `python3.9-devel` and `python3-devel`, but in RHEL8, `python3.9-devel` does not exists and `python3-devel` is for python3.6.